### PR TITLE
Dockerhub Enhancements

### PR DESCRIPTION
**Pull request by petervo**
Adds support for pushing to a fork and multiple docker files.

### CODE REVIEW

File: release/release-dockerhub [hunk 1]
```diff
@@ -29,13 +29,14 @@ SRPM=${RELEASE_SRPM:-$PWD/srpm}
...
 REPO=""
+SOURCE_REPO=""
 WORKDIR=""
 PUSH=""
 CLEANUP=""
 
 usage()
 {
-    echo "usage: release-dockerhub [-qvxz] [-p SRPM] REPO" >&2
+    echo "usage: release-dockerhub [-qvxz] [-p SRPM] [-s SOURCE_REPO ] REPO" >&2
```
Comment (stefwalter) [inline release/release-dockerhub]: Can we make this just be an optional two arguments instead of one?

Comment (stefwalter) [inline release/release-dockerhub]: That would make it much more clear in the script file what is happening.

Comment (petervo) [inline release/release-dockerhub]: We always need a target repo. Having two ways to pass it in seems more confusing to me.

Or did you mean add a -r REPO required argument and remove the $1 REPO argument?

Comment (stefwalter) [inline release/release-dockerhub]: I can make the changes ... one string argument would indicate $SOURCE_REPO == $REPO ... two would mean they are different.

Comment (petervo) [inline release/release-dockerhub]: I pushed a fixup commit without amending in case it conflicts with what you are doing.

**Pull request closed without merging**